### DISCUSSION TITLE
Format to send sensor group tags with FALCON_TAGS env var

### DISCUSSION THREAD

**Discussion by jettore** [CrowdStrike/falcon-scripts#262] [General]
Trying to use the FALCON_TAGS  env var to pass some tags when registering the client but have not been successful.  I'm guessing it's related to the format of the tags.  Can someone share the correct format needed?

**Comment by redhatrises** [comment]
The TAGS variable should follow what is allowed by the `falconctl` utility. So the format for tags is:

```
<comma separated list of tags for sensor grouping> (allowed characters: all alphanumerics, '/', '-', '_', '@', and '.'
```

In other words:

```
export FALCON_TAGS="tag1,tag2"

Generally wouldn't recommend `@` or `.` as part of tags
